code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -116,7 +116,7 @@
 			M.visible_message("<span class='warning'>[M] looks nervous!</span>")
 			M.AdjustConfused(15)
```

Comment:
```suggestion
			M.AdjustConfused(30 SECONDS)
```

You've missed quite a few of these procs here and there. You should do a search to make sure you didn't miss any others.